packages/language-server/server/server.js

```diff
@@ -1,68 +1,32 @@
-const path = require("path");
-const babylon = require("@babel/parser");
-const { getConfig } = require("@hegel/cli/lib/config");
-const { importModule } = require("@hegel/cli/lib/module");
-const { promises: { readFile } } = require("fs");
 const {
-  createGlobalScope,
-  createModuleScope,
   PositionedModuleScope,
-} = require("@hegel/core");
+} = require("@hegel/core/type-graph/module-scope");
```

Comment:
Inside another package, we should use `@hegel/core/index.js` as a point to `@hegel.core`. So, can you please return the previous variant?